lib/cds_importer/entity_mapper.rb

```diff
@@ -60,7 +60,13 @@ def save_record(record)
     end
 
     def remove_excluded_geographical_areas!
-      return if xml_node['sid'].blank?
+      if xml_node['sid'].blank?
+        message = 'Skipping removal of measure geographical exclusions due to missing measure sid.'
+
+        instrument_warning(message, xml_node)
```

Comment:
Is it possible to have some unit tests here to check that the right message and the right xml_node is being passed?